src/config/Subsites/index.js

```diff
@@ -0,0 +1,12 @@
+import loadable from '@loadable/component';
+
+export const loadables = {
+  'light-blue': loadable.lib(() => import('./light-blue')),
```

Comment:
io li chiamerei con il prefisso 'subsite-styles', o qualcosa di simile, in modo da capire che si tratta degli stili dei sottositi. 
Ad esempio 'subsite-styles-light-blue'

Reply:
Ok per me, però poi va cambiato di conseguenza anche qui: 

https://github.com/RedTurtle/design-volto-theme/blob/1627a3aed8134ba27aa0aa5c7251902b61c474c0/src/customizations/volto/components/theme/AppExtras/AppExtras.jsx#L24-L25

procedi pure in autonomia sul branch.